<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>
    MIDI Virtual Piano Keyboard
    Copyright (C) 2008-2020, Pedro Lopez-Cabanillas

    This program is free software; you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation; either version 3 of the License, or
    (at your option) any later version.

    This program is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU General Public License for more details.

    You should have received a copy of the GNU General Public License along
    with this program; If not, see http://www.gnu.org/licenses/
 </comment>
 <class>RiffImportDlg</class>
 <widget class="QDialog" name="RiffImportDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import SoundFont instruments</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../data/vmpk.qrc">
    <normaloff>:/vpiano/vmpk_32x32.png</normaloff>:/vpiano/vmpk_32x32.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelInput">
        <property name="text">
         <string>Input File</string>
        </property>
        <property name="buddy">
         <cstring>m_input</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="m_input">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="whatsThis">
         <string>This text box displays the path and name of the selected SoundFont to be  imported</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="m_inputBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>27</width>
          <height>27</height>
         </size>
        </property>
        <property name="whatsThis">
         <string>Press this button to select a SoundFont file to be imported</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelName">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLabel" name="m_name">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelVersion">
        <property name="text">
         <string>Version</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLabel" name="m_version"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelCopyright">
        <property name="text">
         <string>Copyright</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLabel" name="m_copyright"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelOutput">
        <property name="text">
         <string>Output File</string>
        </property>
        <property name="buddy">
         <cstring>m_output</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="m_output">
        <property name="whatsThis">
         <string>This text box displays the name of the output file in .INS format that will be created</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="m_outputBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>27</width>
          <height>27</height>
         </size>
        </property>
        <property name="whatsThis">
         <string>Press this button to select a path and file name for the output file</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>m_input</tabstop>
  <tabstop>m_inputBtn</tabstop>
  <tabstop>m_output</tabstop>
  <tabstop>m_outputBtn</tabstop>
 </tabstops>
 <resources>
  <include location="../data/vmpk.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RiffImportDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RiffImportDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
